src/agents/common/utils.py

```diff
@@ -0,0 +1,88 @@
+from collections.abc import Sequence
+from typing import Any, Literal
+
+from gen_ai_hub.proxy.langchain import ChatOpenAI
+from langchain.agents import AgentExecutor, OpenAIFunctionsAgent
+from langchain_core.messages import AIMessage, BaseMessage, SystemMessage
+from langchain_core.prompts import MessagesPlaceholder
+from langgraph.constants import END
+
+from agents.common.state import AgentState
+from utils.logging import get_logger
+
+logger = get_logger(__name__)
+
+EXIT = "Exit"
```

Comment:
Can we move all global constants to one file `agents/common/constants.py`? Atm they are all dispersed in different files.